### PR TITLE
IE8+ only edition of Pure with W3C validation fixes

### PR DESCRIPTION
**Pull request by absalomedia**
IE8 rollup fixes & W3C validation fixes including @lkraav work on #551.

Addresses: #575 & #576

Includes all issues resolved on #551 as well as: #381, #480, #504, #539, #583, #584

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by redonkulus** [comment]
Can we remove the bower changes? I would prefer to do that in another PR since the pure site relies on it.

**Comment by redonkulus** [comment]
@absalomedia the commit history is a bit hard as its a combination of yours and @lkraav, while also reverting some changes made previously. If we want to keep the attribution for both of you, maybe we can rebase the commits down to 1 commit for each of you.  Plus, it looks like there are merge conflicts as well that would need to be resolved.

I'd also like to remove the bower changes to a separate PR later on.

File: src/grids/css/grids-core.css [hunk 1]
```diff
@@ -2,9 +2,6 @@
 
 .pure-g {
     letter-spacing: -0.31em; /* Webkit: collapse white-space between units */
-    *letter-spacing: normal; /* reset IE < 8 */
-    *word-spacing: -0.43em; /* IE < 8: collapse white-space between units */
-    text-rendering: optimizespeed; /* Webkit: fixes text-rendering: optimizeLegibility */
```
Comment (redonkulus) [inline src/grids/css/grids-core.css]: why was this removed?

File: src/grids/css/grids-core.css [hunk 2]
```diff
@@ -55,12 +48,9 @@
 
 .pure-u {
     display: inline-block;
-    *display: inline; /* IE < 8: fake inline-block */
-    zoom: 1;
     letter-spacing: normal;
     word-spacing: normal;
-    vertical-align: top;
-    text-rendering: auto;
+    vertical-align: bottom;
```
Comment (redonkulus) [inline src/grids/css/grids-core.css]: what was the reason for changing this one?

Comment (absalomedia) [inline src/grids/css/grids-core.css]: This is #539 (which I replicated in testing) & removing text-rendering as text-rendering isn't W3C valid, even at CSS3 spec.

Comment (absalomedia) [inline src/grids/css/grids-core.css]: @lkraav no it's not connected to text-rendering. `vertical-align` needs to go to the bottom to fix #539

**Comment by redonkulus** [comment]
Lets open a new PR that only includes the changes to remove < IE 8, then the maintainers can evaluate if and when to incorporate this.
